ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -263,6 +263,22 @@ def quantile(input, q, dim=None, keepdim=False, *, interpolation="linear", out=N
 }
 
 
+@numpy_to_torch_style_args
+@to_ivy_arrays_and_back
+def nanquantile(input, q, dim=None, keepdim=False, *, interpolation="linear", out=None):
+    return ivy.nanquantile(
+        input, q, axis=dim, keepdims=keepdim, interpolation=interpolation, out=out
+    )
+
+
+nanquantile.unsupported_dtypes = {

```

Comment:
You're only supposed to point out the unsupported dtypes for this frontend function. (The dtypes not supported by the function of the original framework)
So just `"torch": ("float16", "bfloat16")` would do.

The unsupported dtypes for the backend are determined by the test function itself